projects/angular/storyteller/src/app/routes/user-info/user-info.component.html

```diff
@@ -0,0 +1,24 @@
+<div class="user-info-page">
+  <mat-spinner [diameter]="70" *ngIf="isLoading"></mat-spinner>
+  <div class="align-container" *ngIf="!isLoading">
+    <div class="project-menu">
+      <div class="card menu-card">
+        <img src="/assets/default-avatar.png">
+        <div class="name">{{ user.getFirstName() }} {{ user.getLastName() }}</div>
+        <div class="hr"></div>
+        <div class="email">{{user.getEmail()}}</div>
+      </div>
```

Comment:
I removed social network buttons, since zeplin mock doesn't contain them.
But maybe I don't know something. Please tell me, if it was a mistake.